### PR TITLE
docs(installing-deis): document installation process for alpha

### LINKED ISSUE #6: [install] Document install process for alpha
We need a common location for alpha instructions. Ideally, this would/could use helm, but should include all the info needed to stand up the deis alpha.

Relevant alpha components:
* deis/postgres#5
* deis/etcd#7
* deis/registry#1
* deis/minio#1
* deis/builder

### PR DESCRIPTION
**Pull request by bacongobbler**
closes #6

### CODE REVIEW

File: docs/src/installing-deis/installing-the-deis-platform.md [hunk 1]
```diff
@@ -1,82 +1,34 @@
 # Installing the Deis Platform
 
-We will use the `deisctl` utility to provision the Deis platform from a CoreOS host or a workstation that has SSH access to CoreOS.
+We will use the `helm` utility to provision the Deis platform from a CoreOS host or a workstation
+that has SSH access to CoreOS. If you don't have `helm` installed, see [installing helm][helm] for
+more info.
```
Comment (krancour) [inline docs/src/installing-deis/installing-the-deis-platform.md]: Not sure that OS is a relevant factor here.  Might want to revise this bit.

Comment (bacongobbler) [inline docs/src/installing-deis/installing-the-deis-platform.md]: good catch. I must've glazed over this bit.

**Comment by bacongobbler** [comment]
@krancour I have addressed your comment. Does that look better to you? :)

**Comment by krancour** [comment]
:shipit:

**Pull request merged**